components/remote_ros_connectivity.mdx

```diff
@@ -0,0 +1,52 @@
+To use ROS desktop tools, you will need the Remote PC to be able to connect to your robot's ROS master. This will allow
+you to run ROS commands like `rostopic list`, `rostopic echo`, `rosnode list`, and others, from the Remote PC and the
```

Comment:
Refer to other comments about 'Remote PC'

Reply:
Updated